synapse/lib/stormhttp.py

```diff
@@ -250,6 +250,10 @@ class LibHttp(s_stormtypes.Lib):
                   'returns': {'type': 'str', 'desc': 'The reason phrase for the status code.', }}},
     )
     _storm_lib_path = ('inet', 'http')
+    _storm_lib_perms = (
+        {'perm': ('inet', 'http', 'proxy'), 'gate': 'cortex',
```

Comment:
should discuss if this should be `storm.lib.inet.http.proxy` to be in line with #3135  changes